hack/tools/mdbook/releaselink/releaselink.go

```diff
@@ -49,7 +50,11 @@ func (l ReleaseLink) Process(input *plugin.Input) error {
 
 		gomodule := tags.Get("gomodule")
 		asset := tags.Get("asset")
-		versionRange := semver.MustParseRange(tags.Get("version"))
+		includePrereleases := tags.Get("prereleases") == "true"
```

Comment:
Masterminds/semver skips prereleases in its `Contraints` unless the constraint has the prerelease separator + some value. For example, `>= 0.2.0 < 0.3.0` will only look at 0.2.x, whereas `>= 0.2.0 < 0.3.0-0` will check for prereleases too.

Reply:
I can either switch to the other lib again, or just drop this flag

Reply:
Drop the flag